packages/create-sitecore-jss/src/templates/nextjs-xmcloud/scripts/generate-component-builder/plugins/code-extraction.ts

```diff
@@ -0,0 +1,16 @@
+import { ComponentBuilderPlugin, ComponentBuilderPluginConfig } from '..';
+import { extractComponents } from '@sitecore-jss/sitecore-jss-dev-tools';
+
+/**
+ * If consented to, sends components code for code generation learning to XMC
+ */
+class CodeExtractionPlugin implements ComponentBuilderPlugin {
+  order = 9999;
```

Comment:
`code-extraction` should be executed right after the `component-builder` plugin, correct?
Maybe we can set the `component-builder` plugin order to 9 and assign 10 to `code-extraction`.
At the very least, the order values for these plugins should not be the same.

Reply:
There's really no specific time we need it to execute, outside of "after component builder is generated". For reliability sake (i.e. in case custom plugins are present), we launch extraction at the end of component builder process.